src/exoplanet/light_curves/limb_dark.py

```diff
@@ -55,12 +52,13 @@ def __init__(self, u1, u2=None, model=None):
                 "use `starry` for more flexibility"
             )
             try:
-                assert_op = theano.assert_op.Assert(msg)
+                assert_op = Assert(msg)
             except AttributeError:
-                assert_op = tt.opt.Assert(msg)
+                # TODO: No longer exists. Should this just be removed? Seems handled by compat
+                assert_op = pt.opt.Assert(msg)
```

Comment:
`pt.opt.Assert` no longer exist in PyTensor. This seems covered by the try-except for theano in `compat` so I guess it can be removed?

Reply:
Are you saying that the `Assert` from compat should just work? Or that there is no longer an assert op in Pytensor?

Reply:
Yes, I think `Assert` from compat should just work. In PyTensor `Assert` is in the `raise_op` module.